EmployeeManagementService/EmployeeManagementService.Infrastructure/Persistence/EmployeeRepository.cs

```diff
@@ -0,0 +1,20 @@
+﻿using EmployeeManagementService.Infrastructure.Persistence.Entities;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace EmployeeManagementService.Infrastructure.Persistence
+{
+    public class EmployeeRepository
+    {
+        public async Task<List<Employee>> GetAllEmployees()
+        {
+            using (var context = new RofSchedulerContext())
+            {
+                return await context.Employees.ToListAsync();

```

Comment:
This returns all the fields of the Employees entities.

The enhancement asked for you to return the fields FirstName, LastName. Role, and Username only.

Reply:
Fixed